test/functorch/test_aotdispatch.py

```diff
@@ -2789,34 +2789,34 @@ def forward(self, add, tangents_1):
     return (mul_1, None)""",
         )
 
-    def test_backward_mutation_metadata(self):
-        class BwMutation(torch.autograd.Function):
-            @staticmethod
-            def forward(ctx, a, b):
-                ctx.save_for_backward(b)
-                return a.clone(), b.clone()
-
-            @staticmethod
-            def backward(ctx, grad_a, grad_b):
-                (b,) = ctx.saved_tensors
-                # bw metadata mutation
-                b.transpose_(1, 0)
-                return grad_a.clone(), grad_b.clone()
+    # def test_backward_mutation_metadata(self):
```

Comment:
Are you giving up on this test (and test/dynamo_expected_failures/TestAutograd.test_custom_function_local_inplace and test/dynamo_expected_failures/TestAutograd.test_custom_function_saved_tensors ) for now?

Reply:
Ditto for test_custom_function_saved_tensors, it's grad_fn attribute access

Reply:
Are you going to fix these things before 2.10? If not we're going to need to mark this PR as BC-breaking

Reply:
I am not planning to ... I dont think these are BC breaking though. The issues still exist on the main, just that Dynamo tracing changes slightly to cause a graph break at a different place.

Reply:
The tests used to succeed, and now they fail right? That seems bc-breaking to me.